corehq/apps/hqwebapp/utils.py

```diff
@@ -264,12 +260,6 @@ def sidebar_to_dropdown(sidebar_items, domain=None, current_url_name=None):
           'html': None,
           'is_divider': False,
           'is_header': False,
-          'title': u'New Mobile Worker',
-          'url': '/a/sravan-test/settings/users/commcare/add_commcare_account/'},
```

Comment:
Haha, you're absolutely right, @sravfeyn - I didn't zoom out the diff enough to see that.  Looks like this part should probably stay as-is, except maybe with the URL name and output URL updated (not that that's necessary either).

I'm not going to block this PR on that, but maybe @biyeun or @sravfeyn should revert that part in another PR.